plugins/modules/aci_fabric_span_src_group_src_node.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Akini Ross <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_fabric_span_src_group_src_node
+short_description: Manage Fabric SPAN source nodes (span:RsSrcToNode)
+description:
+- Manage Fabric SPAN source nodes on Cisco ACI fabrics.
+- Source nodes are only configurable when the C(source_group) provided has C(drop_packets) set to I(true).
```

Comment:
```suggestion
- Source nodes are only configurable when the I(source_group) provided has I(drop_packets) set to C(true).
```
drop_packets - is not part of the current module?

Reply:
correct it is part of the module aci_fabric_span_src_group